cv_bridge/include/cv_bridge/cv_bridge.h

```diff
@@ -93,6 +98,9 @@ class CvImage
    */
   void toImageMsg(sensor_msgs::Image& ros_image) const;
 
+  void toCompressedImageMsg(sensor_msgs::CompressedImage& ros_image, const std::string& dst_format = std::string()) const;
```

Comment:
it a string that you can compressed for format opencv has suport.
even in there code document they don't show OpenCv const or Reference. 
[opencv doc][1]:

[1]:http://docs.opencv.org/modules/highgui/doc/reading_and_writing_images_and_video.html#imencode